documentation/userguide.md

```diff
@@ -95,12 +95,39 @@ After restarting the IIS/recycling the pool and opening the application in a bro
 
 ### Azure Cloud
 
-Azure instantly kills the application when it is stopped/restarted, thus the profiler has no time to write anything to the trace file. To remedy this, enable eager mode (see the Configuration section. Please note that eager mode should only be used together with light mode as it will otherwise drastically increase lag during the application startup.
-
-To deploy the profiler to the Azure cloud, enable the FTP account. Store the profiler in a directory that will not be overwritten by a new deployment, e.g. `D:\home\site\repository\profiler\`
-Also create a new directory to write the traces to, e.g. `D:\home\site\repository\traces\`
-
-To set the environment variables in Azure, go to the application settings and add them as new application settings entries.
+If the application is running in an Azure App Service, the following steps need to be taken:
+
+**Install the Profiler:**
+To deploy the profiler, [enable the FTP account](https://docs.microsoft.com/en-us/azure/app-service/app-service-deploy-ftp) and upload the profiler DLLs to a location that will not be overwritten by a new app deployment, e.g., `D:\home\site\repository\profiler\`.
```

Comment:
Your links seem to be broken in the final PDF:

![selection_350](https://user-images.githubusercontent.com/38839/49721611-fe67a500-fc62-11e8-84b7-1ac5057d8412.png)

Please fix.
I'd also prefer not inlining these links as it makes the document hard to read in source form. I'd prefer

    [link text][short-name]

and then at the end of the document

    [short-name]: http://link

Reply:
I wasn't aware there's a PDF generated... 0:-) How can I generate it to check? The links are fine by themselves.

Reply:
Ah, found `npm markdown-pdf`. Will try this out.

Reply:
@karottenreibe I think there's still the link URL written out in the PDF. Should I apply the change mentioned above to make them disappear?

Reply:
yes please :-)